package.json

```diff
@@ -33,13 +33,14 @@
     "lint-staged": "^16.2.1",
     "prettier": "^3.6.2",
     "ts-jest": "^29.4.4",
-    "tsx": "^4.20.6",
+    "tsx": "^4.20.5",
     "turbo": "^2.5.6",
     "typescript": "^5.9.2"
   },
   "workspaces": [
     "apps/*",
-    "packages/*"
+    "packages/*",
+    "mcp-agent-server"
```

Comment:
@CharlieHelps oops please move this whole package into the `packages/` directory for now, I didn't realize this when I wrote the original issue